lib/adapters/translate_encoder.js

```diff
@@ -73,13 +72,12 @@ var encoder = processor({
       for (var idx = 0; idx < vals.length; idx += 2) {
         var k = encoder(vals[idx]);
         var v = encoder(vals[idx + 1]);
-        if (k instanceof AMQPSymbol) {
+        result[k] = v;
```

Comment:
I think the reason I had this was that I couldn't have object keys (e.g. keys which where themselves objects). Apparently I was wrong?